From: Will Thompson <wjt@endlessm.com>
Date: Fri, 10 Jul 2020 16:05:41 +0100
Subject: Revert "Add the directory option to the file chooser impl interface"

This reverts commit ee041ae8c8c8b0534bdd300953f131f1c93434cd.
Documentation for this parameter was already added in c616b491 so this
just duplicates it.

Origin: upstream, 1.7.3, commit:b6baf10ab84e3c1227abc714a66fe327b111a9c2
---
 data/org.freedesktop.impl.portal.FileChooser.xml | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/data/org.freedesktop.impl.portal.FileChooser.xml b/data/org.freedesktop.impl.portal.FileChooser.xml
index 8c684b0..3a7174d 100644
--- a/data/org.freedesktop.impl.portal.FileChooser.xml
+++ b/data/org.freedesktop.impl.portal.FileChooser.xml
@@ -65,12 +65,6 @@
               Whether to select for folders instead of files. Default is to select files.
             </para></listitem>
           </varlistentry>
-          <varlistentry>
-            <term>directory b</term>
-            <listitem><para>
-              Whether to select for folders instead of files. Default is to select files.
-            </para></listitem>
-          </varlistentry>
           <varlistentry>
             <term>filters a(sa(us))</term>
             <listitem><para>
